pkg/cincinnati/cincinnati.go

```diff
@@ -243,6 +241,15 @@ func (c Client) GetVersions(ctx context.Context, uri *url.URL, channel string) (
 	return Vers, nil
 }
 
+func getSemVer(stringVers []string) semver.Version {
```

Comment:
```suggestion
func getLatestSemVer(stringVers []string) semver.Version {
```

Reply:
Also check `len(stringVers) == 0` so indexing never panics.

Reply:
This has been fixed on the branch so I am going to resolve this thread.